packages/shared/components/modals/AccountActionsMenu.svelte

```diff
@@ -17,8 +17,8 @@
         modal.close()
     }
 
-    function handleViewStorageDepositClick(): void {
-        openPopup({ id: 'storageDepositBreakdown' })
+    function handleViewBalanceClick(): void {
```

Comment:
This is a new convention we're working towards for event handlers.

```suggestion
    function onViewBalanceClick(): void {
```